CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- Changed the scorecard basic test, `Writing into CRs has an effect`, to include the http.MethodPatch as part of its test criteria alongside http.MethodPut and http.MethodPost. ([#2509](https://github.com/operator-framework/operator-sdk/pull/2509))
+
```

Comment:
This is not the correct CHANGELOG entry for this bug.

Reply:
Ops. I did it pass. tks for the catch. I will fix.